src/remote/utils.go

```diff
@@ -199,7 +199,13 @@ func (c *Client) locallyCacheResults(target *core.BuildTarget, digest *pb.Digest
 	}
 	data, _ := proto.Marshal(ar)
 	metadata.RemoteAction = data
-	c.state.Cache.Store(target, c.localCacheKey(digest), nil)
+	// TODO(jpoole): Similar to retrieveTargetMetadataFromCache, it would be cleaner if we could store
+	//               into the cache from an arbitrary reader.
+	if err := build.StoreTargetMetadata(target, metadata); err != nil {
+		log.Warning("%s", err)
```

Comment:
Does this log line work? We need Warningf or just warning without the format string.

Reply:
Yep it does... remember this is not the same log library we use internally at TM, it always interpolates.